docs/contribute-how-to.md

```diff
@@ -5,10 +5,242 @@ mentions:
 -   TheItsNameless
```

Comment:
The full front-matter is this:

```
---
title: How to contribute to our wiki
hidden: true
mentions:
-   TheItsNameless
---
```

Looks correct for me, maybe GitHub made it look weird because it only shows the parts that got changed and not the parts that didn't.